### DISCUSSION TITLE
検索オプションについて質問

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#517] [要望・アイデア・不具合]
ご担当者　様

記事検索を行う際に、101ページを見ようとすると404エラーが変えってくるのですが、何か制限をかけていますでしょうか？

例えば、以下の`created:>=YYYY-MM-DD`オプションは効きません。`stocks:>N`も同様です。
ご確認お願いします。

- [created:>=2023-01-01](https://qiita.com/search?q=created%3A%3E%3D2023-01-01&sort=&stocked=&page=101)

![スクリーンショット 2023-08-03 21 33 07](https://github.com/increments/qiita-discussions/assets/38848514/67de3f72-f224-474c-94d8-4d7d1df5ffc6)

**Comment by wataru86** [comment]
@Brutus03

ご不便をおかけして申し訳ありません🙏
検索システムのパフォーマンスの都合上、現在100ページまでに制限しています。
検索結果が100ページ以内になるように、絞り込みをしていただけると幸いです。
今後、101ページ以上は見れない事が伝わるような修正を予定しています🙏

**Comment by Brutus03** [comment]
> 今後、101ページ以上は見れない事が伝わるような修正を予定しています🙏

承知しました。

**Reply by P-SiZK** [reply]
@Brutus03 
お待たせしました 🙇 

100 ページを超えて検索した際の表示内容を変更しました！
下記の画像のように、100 ページまでの制限があることが分かるようにしました。

![](https://github.com/increments/qiita-discussions/assets/54477535/9f692e79-c8c2-41b2-8bab-f2e2af419f74)

**Reply by Brutus03** [reply]
@P-SiZK 
ご対応ありがとうございます。
修正を確認したため、本件クローズで大丈夫です。
